src/integrationtests/java/com/ericsson/ei/artifactintegrationtests/ArtifactFlowStepsIT.java

```diff
@@ -0,0 +1,165 @@
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootContextLoader;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestExecutionListeners;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.context.support.DependencyInjectionTestExecutionListener;
+
+import com.ericsson.ei.App;
+import com.ericsson.ei.erqueryservice.ERQueryService;
+import com.ericsson.ei.mongodbhandler.MongoDBHandler;
+import com.ericsson.ei.utils.HttpRequest;
+import com.ericsson.ei.utils.HttpRequest.HttpMethod;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import util.IntegrationTestBase;
+
+@Ignore
+@RunWith(SpringRunner.class)
+@SpringBootTest(classes = App.class, webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+@ContextConfiguration(classes = App.class, loader = SpringBootContextLoader.class)
+@TestExecutionListeners(listeners = { DependencyInjectionTestExecutionListener.class })
+public class ArtifactFlowStepsIT extends IntegrationTestBase {
+    private static final Logger LOGGER = LoggerFactory.getLogger(ArtifactFlowStepsIT.class);
+
+    private static final String UPSTREAM_INPUT_FILE = "src/test/resources/upStreamInput.json";
+    private static final String RULES_FILE_PATH = "src/test/resources/ArtifactRules_new.json";
+    private static final String EVENTS_FILE_PATH = "src/test/resources/test_events.json";
+    private static final String AGGREGATED_OBJECT_FILE_PATH = "src/test/resources/AggregatedDocumentInternalCompositionLatest.json";
+    private static final String AGGREGATED_OBJECT_ID = "6acc3c87-75e0-4b6d-88f5-b1a5d4e62b43";
+    private static final String SUBSCRIPTIONS_PATH = "src/integrationtests/resources/subscriptionsForArtifactTest.json";
+    private static final String host = "localhost";
+
+
+    private long startTime;
+    private static ObjectMapper objectMapper = new ObjectMapper();
+
+    @Value("${server.port}")
+    int port;
+
+    @Mock
+    private ERQueryService erQueryService;
+
+    @Autowired
+    private RabbitTemplate rabbitMqTemplate;
+
+    @Autowired
+    MongoDBHandler mongoDBHandler;
+
+    @Given("^subscriptions are uploaded$")
+    public void subscriptions_are_created() throws URISyntaxException, IOException {
+        startTime = System.currentTimeMillis();
+
+        URL subscriptionsInput = new File(SUBSCRIPTIONS_PATH).toURI().toURL();
+        ArrayNode subscriptionsJson = (ArrayNode) objectMapper.readTree(subscriptionsInput);
+
+        HttpRequest getRequest = new HttpRequest(HttpMethod.POST);
```

Comment:
getRequest should be postRequest I guess

Reply:
done